__tests__/String/last.test.ts

```diff
@@ -0,0 +1,10 @@
+import { S, O } from '../..'
+
+describe('last', () => {
+  it('returns None if the given string is empty.', () => {
+    expect(S.last('')).toEqual(O.None)
+  })
+  it('returns Some(value) where value is the last character of the string', () => {
+    expect(S.last('last-test')).toEqual('t')
+  })
+})
```

Comment:
could you please add a test with the `*` string as a first argument of `it`? (it automatically adds an example in the documentation)